### DISCUSSION TITLE
Add chromecast/google cast support

### DISCUSSION THREAD

**Discussion by ippocratis** [savvasdalkitsis/uhuruphotos-android#141] [Ideas]
Well thats tricky as idk how Foss you want to keep the app
cast  is cool to have to send media on the big screen but it is a proprietary google protocol
Also idk  how hard it is to implement

**Comment by savvasdalkitsis** [comment]
I'm not that fussed about pure foss tbh. I had this in the back of my head an an option so this is definitely one for the blacklog
